picard/cluster.py

```diff
@@ -70,7 +70,46 @@
 from picard.ui.item import Item
 
 
-class Cluster(QtCore.QObject, Item):
+class FileList(QtCore.QObject, Item):
+
+    def __init__(self, files=None):
+        QtCore.QObject.__init__(self)
+        self.metadata = Metadata()
+        self.orig_metadata = Metadata()
+        self.files = files or []
+        self.update_metadata_images_enabled = True
+        for file in self.files:
+            if self.can_show_coverart:
+                file.metadata_images_changed.connect(self.update_metadata_images)
+        update_metadata_images(self)
+
+    def iterfiles(self, save=False):
```

Comment:
`save` parameter isn't used, is there a reason to keep it?

Reply:
Yes, this is for API compatibility with `Album` and `ClusterList`.